packages/expo-notifications/src/getExpoPushTokenAsync.ts

```diff
@@ -32,6 +32,8 @@ export default async function getExpoPushTokenAsync(options: Options = {}): Prom
 
   const experienceId =
     options.experienceId ||
+    Constants.manifest2?.extra?.eas?.projectId ||
+    Constants.manifest?.projectId ||
```

Comment:
The project ID should be treated separate from the experience ID. Specifically, the HTTP request to the server should send a different field (projectId).

Reply:
Oh man, nice catch. Now I'm questioning whether my test plan actually worked (the `yarn link`).

Reply:
The server might have accepted the value but I wouldn't have expected a notification to have worked. Might be worth checking with a network proxy to see what the real request payload is.

Reply:
This ended up being insanely hard to test because it needed a `projectId`, which at the moment is only served by the server. So I needed to publish an experience that uses a the modified `expo-notifications` (with the change). I updated the test plan to test this.